tests/wen_new_standard.test.ts

```diff
@@ -0,0 +1,687 @@
+        expect(groupAccount.updateAuthority).to.eql(groupAuthorityPublicKey);
+      });
+
+      it("should reference the group mint", async () => {
+        expect(groupAccount.mint).to.eql(groupMintPublicKey);
+      });
+
+      it("should have a max size", async () => {
+        expect(groupAccount.maxSize).to.eql(createGroupArgs.maxSize);
+      });
+
+      it("should have no members denoted by a size of 0", async () => {
+        expect(groupAccount.size).to.eql(0);
+      });
+    });
+
+    describe("after updating", () => {});
+
+    describe("after adding a mint as a member", () => {
+      const mintKeyPair = Keypair.generate();
+
+      const mintAuthPublicKey = provider.wallet.publicKey;
+      const mintPublicKey = mintKeyPair.publicKey;
+      const mintTokenAccount = getAssociatedTokenAddressSync(
+        mintPublicKey,
+        mintAuthPublicKey,
+        false,
+        TOKEN_2022_PROGRAM_ID
+      );
+
+      const [member] = PublicKey.findProgramAddressSync(
+        [MEMBER_ACCOUNT_SEED, mintPublicKey.toBuffer()],
+        program.programId
+      );
+
+      let memberAccountInfo: AccountInfo<Buffer>;
+      let memberAccount;
+
+      describe("the mint", () => {
+        before(async () => {
+          const createMintAccountIx = await program.methods
+            .createMintAccount({
+              permanentDelegate: null,
+              name: faker.lorem.text(),
+              symbol: faker.lorem.word(),
+              uri: faker.internet.url(),
+            })
+            .accounts({
+              authority: mintAuthPublicKey,
+              mint: mintPublicKey,
+              mintTokenAccount,
+              payer: mintAuthPublicKey,
+              receiver: mintAuthPublicKey,
+            })
+            .instruction();
+
+          await program.methods
+            .addMintToGroup()
+            .accountsStrict({
+              authority: groupAuthorityPublicKey,
+              group,
+              mint: mintPublicKey,
+              payer: mintAuthPublicKey,
+              manager,
+              member,
+              systemProgram: SystemProgram.programId,
+              tokenProgram: TOKEN_2022_PROGRAM_ID,
+            })
+            .preInstructions([createMintAccountIx])
+            .signers([mintKeyPair, groupAuthorityKeyPair])
+            .rpc({
+              skipPreflight: true,
+              preflightCommitment: "confirmed",
+              commitment: "confirmed",
+            });
+
+          memberAccountInfo =
+            await program.account.tokenGroupMember.getAccountInfo(member);
+          memberAccount = program.coder.accounts.decode(
+            "tokenGroupMember",
+            memberAccountInfo.data
+          );
+        });
+
+        it("should be an account owned by the program", async () => {
+          expect(memberAccountInfo.owner).to.eql(program.programId);
+        });
+        it("should point back to the group", async () => {
```

Comment:
Can we also verify mint and member_number here as well. Just to make sure these are being added properly.